velox/experimental/cudf/exec/CudfFilterProject.h

```diff
@@ -67,18 +67,49 @@ class CudfFilterProject : public exec::Operator, public NvtxHelper {
   }
 
  private:
+  // Copied from operator FilterProject.
+  void initializeFilterProject();
+
+  /// Data for accelerator conversion.
+  struct Export {
+    const exec::ExprSet* exprs;
+    bool hasFilter;
+    const std::vector<exec::IdentityProjection>* resultProjections;
+  };
+
+  Export exprsAndProjection() const {
+    return Export{exprs_.get(), hasFilter_, &resultProjections_};
+  }
+
   bool allInputProcessed();
+
   // If true exprs_[0] is a filter and the other expressions are projections
   const bool hasFilter_{false};
+  const bool lazyDereference_;
+
+  std::unique_ptr<exec::ExprSet> exprs_;
+  int32_t numExprs_;
```

Comment:
They don't appear to be used after initialize. They can be removed after consolidating the `initialize` and `initializeFilterProject` methods.